txrelayer/txrelayer.go

```diff
@@ -159,7 +159,7 @@ func (t *TxRelayerImpl) waitForReceipt(hash ethgo.Hash) (*ethgo.Receipt, error)
 			return receipt, nil
 		}
 
-		if count > 100 {
+		if count > 1000 {
```

Comment:
Maybe we should put this value to constant?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/8de4d6a725d75f5819e1071cfac7a5ffbab412fb